pages/api/user/[address].ts

```diff
@@ -0,0 +1,31 @@
+import { NextApiRequest, NextApiResponse } from "next";
+import models from "@db/models";
+
+async function remove(req: NextApiRequest, res: NextApiResponse) {
+  const { address } = req.query;
```

Comment:
query needs more information provided so it can't be queried by bad actors. We should receive a combo of address+github[name/login/or both]

Reply:
- if user has locked ORACLES amount DENY removal